test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -90,6 +93,14 @@ func describeQatDpdkPlugin() {
 			}
 		})
 
+		ginkgo.AfterEach(func() {
+			ginkgo.By("undeploying DSA plugin")
```

Comment:
```suggestion
			ginkgo.By("undeploying QAT plugin")
```

Reply:
Ah, thanks. I chnaged others, but forgot for QAT. Let me fix them.